src/canmatrix/xlsx.py

```diff
@@ -34,7 +34,7 @@
 import xlsxwriter
 from canmatrix.xls_common import *
 
-
+logger = logging.getLogger(__name__)
```

Comment:
Do we need this if it's not being used?  I think it might be better not to preemptively add it.

Reply:
The logger was used in the xlsx.py file, but the instance came from `from .canmatrix import *` what is wrong. Now it uses own module logger.

Reply:
ack!  new item for the refactor list, no `*` imports.  :]